scripts/init.sh

```diff
@@ -154,37 +147,28 @@ case $1 in
             exit 1
         fi
 
-        log_success_msg "Starting the process" "$name"
+        log_success_msg "Starting the process" "$NAME"
         if which start-stop-daemon > /dev/null 2>&1; then
-            start-stop-daemon --chuid $GROUP:$USER --start --quiet --pidfile $pidfile --exec $daemon -- -pidfile $pidfile -config $config $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &
+            start-stop-daemon --chuid $GROUP:$USER --start --quiet --pidfile $PIDFILE --exec $DAEMON -- -pidfile $PIDFILE -config $CONFIG $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &
         else
-            su $USER -c "nohup $daemon -pidfile $pidfile -config $config $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &"
-        fi
-
-        wait_for_startup && is_process_running
-        if [ $? -ne 0 ]; then
-            log_failure_msg "$name process failed to start"
-            exit 1
-        else
-            log_success_msg "$name process was started"
-            exit 0
+            nohup $DAEMON -pidfile $PIDFILE -config $CONFIG $INFLUXD_OPTS >>$STDOUT 2>>$STDERR &
```

Comment:
May need another `which daemon` check here and if available, do something like...
```
daemon --user=$USER:$GROUP --pidfile=$PIDFILE ...
```
Then fall back to starting $DAEMON directly if `daemon` function isn't available.  `su` may have problems if $USER is the same as the user running the init script.